application/actions/auth.go

```diff
@@ -222,7 +222,7 @@ func authRequest(c buffalo.Context) error {
 	orgID := org.UUID.String()
 	c.Session().Set(OrgIDSessionKey, orgID)
 
-	sp, err := org.GetAuthProvider()
+	sp, err := org.GetAuthProvider(authEmail)
```

Comment:
The param is also needed on lines 272 and 402. I'm thinking that for line 402, you will need to first get the authEmail - maybe the way it's done in `authCallback()`